docs/UserGuide.md

```diff
@@ -252,7 +261,7 @@ Sorts a list of all flashcards according to the criteria given.
 Format: `sort <success|reviewed> <-a|-d>`
 
 Examples: 
-* sort followed by:
+* `sort` followed by:
```

Comment:
do you think it will be better to just remove the “followed by” here also? coz I will be fixing that in #145